src/test/scala/org/broadinstitute/dsde/workbench/sam/openam/PostgresAccessPolicyDAOSpec.scala

```diff
@@ -105,7 +105,18 @@ class PostgresAccessPolicyDAOSpec extends FreeSpec with Matchers with BeforeAndA
               }
             }
 
-            "has the same ActionPatterns with modified descriptions" in pending
+            "has the same ActionPatterns with modified descriptions" in {
+              val myActionPatterns = actionPatterns + ResourceActionPattern("coolNewPattern", "I am the coolest pattern EVER!  Mwahaha", true)
+              val myResourceType = resourceType.copy(actionPatterns = myActionPatterns)
+
+              val myActionPatternsNew = actionPatterns + ResourceActionPattern("coolNewPattern", "I am the NEWEST pattern EVER!  Mwahaha", true)
+
+              val myUpdatedResourceType = myResourceType.copy(actionPatterns = myActionPatternsNew)
+
+              dao.createResourceType(myResourceType).unsafeRunSync() shouldEqual myResourceType
+
+              dao.createResourceType(myUpdatedResourceType).unsafeRunSync() shouldEqual myUpdatedResourceType
```

Comment:
It's too bad there's no way to load the resource type and check that it actually got changed in the database the way we wanted it to

Reply:
forreal. we should add that as new/diverged functionality once opendj is retired as primary DB